test/parallel/test-http-outgoing-renderHeaders.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const common = require('../common');
+const assert = require('assert');
+
+const outHeadersKey = require('internal/http').outHeadersKey;
```

Comment:
It’s read by the test runner, so if you run it as `make test` or `python tools/test.py parallel/test-http-outgoing-renderHeaders`, it will automatically be used. If you want to run the test file manually as `./node test/parallel/test-http-outgoing-renderHeaders.js`, you’ll also need to add the flag yourself.